packages/ui/src/utils/Svgs.ts

```diff
@@ -451,3 +451,77 @@ export const DEFI_IMG = svg`
     </defs>
   </svg>
 `
+
+export const ETH_LOGO = svg`
```

Comment:
For all new svg's could you please run them through https://jakearchibald.github.io/svgomg/ and cleanup unecessary data i.e. `fill="none" xmlns="http://www.w3.org/2000/svg"` as these are inlined into final bundle.

Reply:
Fixed.